### ISSUE TITLE
Is there a way to put the search URL in a sub config for a package?

### ISSUE THREAD

**Issue by bethlakshmi** [yourlabs/django-autocomplete-light#1197]
I'm building a non-admin UI for an app that resides within the Django CMS system.  My code is in it's own, isolated package, called "inventory", and my GUI URLs all use their own urlconf.  I'd really like to situate the URL for the django-autocomplete-light within that sub config, but it seems like the only urlconf that works is the root.  I read the code, and it looks like there's no way to pass a "urlconf" parameter to the widget, the only URL look up call that happens is "reverse(url)" with no other args.

Please let me know if there is a way to do it.
If not, please consider this a feature request.

I'd be up for implementing it, if you can point me to your coding standards/PR process?

**Comment by jpic** [comment]
Does it work when you try this manual ?
https://docs.djangoproject.com/en/3.1/topics/http/urls/#including-other-urlconfs

**Comment by bethlakshmi** [comment]
Ah.  I was using name space urls to keep them distinct.  So it's the fact that the reverse needs to reference the app_name that is the blocker.

**Comment by jpic** [comment]
Nice, good to close then ?

**Comment by bethlakshmi** [comment]
No... I really can't remove the namespace easily and isolating this way is a legitimate use case in django.  Is there a way to specify this when setting up the form widget.
